packages/compiler/package.json

```diff
@@ -22,25 +22,26 @@
         "@types/kss": "^3.0.2",
         "glob": "^10.3.10",
         "js-yaml": "^4.1.0",
-        "kss": "^3.1.0",
-        "marked": "^4.2.12",
-        "typedoc": "~0.19.2",
-        "yargs": "^17.7.1"
+        "kss": "^3.0.1",
+        "marked": "^4.0.12",
+        "tsconfig-resolver": "^3.0.1",
+        "typedoc": "~0.25.2",
+        "yargs": "^17.4.0"
     },
     "devDependencies": {
-        "@types/glob": "^8.0.1",
+        "@types/glob": "^7.2.0",
```

Comment:
ooc why did you have to downgrade?

Reply:
hmm, I think this was a bad rebase/merge since this branch was originally started a long time ago. good catch -- will fix